assets/app/index.html

```diff
@@ -65,7 +65,9 @@
               <li class="divider"></li>
               -->
               <li>
-                <a href="logout">Log out</a>
+                <!-- Stop event propagation to avoid a runtime error when the
+                     dropdown tries to close itself as we're navigating away. -->
+                <a href="logout" ng-click="$event.stopPropagation()">Log out</a>
```

Comment:
If we are lucky, maybe this is already fixed when we move up to the new patternfly version which brings in a newer bootstrap.  Should retest after we merge the update.